app/models/keyword.rb

```diff
@@ -4,6 +4,9 @@ class Keyword < ApplicationRecord
   enum status: { pending: 'pending', processing: 'processing', succeeded: 'succeeded', failed: 'failed' }
 
   has_many :keyword_search_entries, dependent: :destroy
+  has_one :keyword_stats, dependent: :nullify
```

Comment:
How will this be used? I am unsure eif this is needed 🤔

Reply:
Oops, forgot to remove that. Fixed in e8e11f81b5d7df46200e5b567573eccb30b54a62